test/spec/ol/tilerange.test.js

```diff
@@ -58,9 +58,11 @@ describe('ol.TileRange', function() {
     });
 
     describe('with mixed z', function() {
-      expect(function() {
-        return new ol.TileRange.boundingTileRange([3, 1, 3], [4, 2, 0]);
-      }).to.throwException();
+      it('returns the expected TileRange', function() {
```

Comment:
Nice catch.  This one has been around for a while (589c2a85054cffa59f45cabd7c45abf565c2c69e)!